docs/processing/qaqc-criteria-preprocessed.md

```diff
@@ -1,83 +1,124 @@
-  If it is not the case, tweak the brain mask estimation and re-run *fMRIPrep*.
-- [ ] Check that the brain mask does not include parts that are clearly NOT brain.
-  Typically, this appears as bumps surrounding high-intensity areas of signal outside the brain (e.g dura or skull).
-  If it is not the case, tweak skull-stripping and re-run *fMRIPrep*.
 - [ ] Assess the quality of the segmentation:
     - [ ] Check that the blue line correctly outlines the boundary between GM and WM.
     - [ ] Check that the magenta line outlines ventricles.
     - [ ] Check that the contours do not exclude single voxels within piecewise-smooth regions (generally more identifiable in the WM and inside the ventricles).
-      Careful if those single excluded voxels only appear close to tissue boundaries, they are likely misclassified because of partial volume effect, as such it does not consistitute an exclusion criteria.
-    - [ ] Check that the contours do not include other tissues than the tissue of interest (GM; magenta, WM; blue). 
-    - [ ] If you found any of the issues above, tweak the segmentation and re-run *fMRIPrep*.
+      Be careful if those single excluded voxels only appear close to tissue boundaries.
+      They are likely misclassified because of the partial volume effect, and as such, they do not constitute an exclusion criteria.
+    - [ ] Check that the contours do not include other tissues than the tissue of interest (GM; magenta, WM; blue).
+    - [ ] **Do not** use the CompCor confounds for nuisance regression if you find any of the issues above.
 
 ### Spatial normalization of the anatomical T1w reference
 
-- [ ] In order of importance, the following structures should be correctly aligned: 1. ventricles, 2. subcortical regions, 3. corpus callosum, 4. cerebellum, 5. cortical GM.
+- [ ] In order of importance, the following structures should be correctly aligned: (i) ventricles, (ii) subcortical regions, (iii) corpus callosum, (iv) cerebellum, (v) cortical GM.
     Tweak the spatial normalization and re-run *fMRIPrep* in case of a misalignment of the ventricles, subcortical regions, or the corpus callosum.
     You can, however, be more lenient with GM alignment.
 - [ ] Check for severe stretching or distortion of the T1w.
-  If observed, tweak the normalization and re-run *fMRIPrep*.
-
-### Surface reconstruction
-
-!!! note "If you used the `--fs-no-reconall` flag to skip surface-based preprocessing, the section of the report will not exist"
-
-- [ ] Check that the WM-GM boundary outlines (blue line) matches the boundary observed in the underlying image.
-- [ ] Verify that the WM and pial surface do not cross or overlap each other.
-- [ ] Verify that the pial surface (red line) does not extend past the actual pial boundary.
 
-!!! Tip "Evaluating the quality of brain surfaces"
+  !!! note "We can afford to work in subject space because our study is based on a single subject"
 
```

Comment:
Let's remove this comment:

```suggestion
```

(i) This is not really a QA/QC criteria, but rather a study design consideration regarding the analysis.
(ii) Being a design consideration, the language of "we can afford" is inappropriate. Design decisions are based on what you want and need to do to achieve something the best way possible. Informally, this can be okay, but writing it in the protocol is a bit misfortunate.